src/api/SFA.DAS.Provider.Events.Api/Controllers/TransfersController.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Web.Http;
+using MediatR;
+using NLog;
+using SFA.DAS.Provider.Events.Api.Plumbing.WebApi;
+using SFA.DAS.Provider.Events.Api.Types;
+using SFA.DAS.Provider.Events.Application.Data;
+using SFA.DAS.Provider.Events.Application.Period.GetPeriodQuery;
+using SFA.DAS.Provider.Events.Application.Transfers.GetTransfersQuery;
+using SFA.DAS.Provider.Events.Application.Validation;
+
+namespace SFA.DAS.Provider.Events.Api.Controllers
+{
+    [AuthorizeRemoteOnly(Roles = "ReadPayments")]
+    public class TransfersController : ApiController
+    {
+        private const int PageSize = 10000;
+        private readonly ILogger _logger;
+        private readonly IMediator _mediator;
+
+        public TransfersController(ILogger logger, IMediator mediator)
+        {
+            _logger = logger;
+            _mediator = mediator;
+        }
+
+        [HttpGet]
+        [Route("api/transfers", Name = "GetTransfers")]
+        public async Task<IHttpActionResult> GetTransfers(string periodId = null, long? senderAccountId = null, long? receiverAccountId = null, int page = 1)
+        {
+            try
+            {
+                if (!string.IsNullOrEmpty(periodId))
+                {
+                    var period = await GetPeriodAsync(periodId).ConfigureAwait(false);
```

Comment:
This method of calling GetPeriodAsync() is not used at all in the Payments controller. Does this call to the Periods Table add value or are we just getting a period name from it? 

Also since I think both these methods will be called together (I'm guessing a little). Shouldn't the signature and filtering logic be equivalent?

Reply:
I think in the Payments Controller. If no period is passed in it returns all records